docs/content/development/dist-a-mixin.md

```diff
@@ -21,7 +21,7 @@ systems:
 * GOOS: windows, linux, darwin
 * GOARCH: amd64
 
-If you are creating your mixin in Go, you may find the [Makefile][mk] that we use
+If you are creating your mixin in Go, you may find the [Makefile](TODO) that we use
```

Comment:
I don't know what this is referencing, ideas?

Reply:
Sub Makefile -> Magefile and so I would link them[ here I guess](https://github.com/getporter/skeletor/blob/main/magefile.go) 🤔